README.md

```diff
@@ -712,126 +712,126 @@ By index (be careful with the range):
 AttachmentSet ats = re.attachmentSet();
 ```
 
-* Get the Attachment from AttachmentSet
+* Get the attachment from the AttachmentSet:
 
-By index
+By index:
   
 ```java
 Attachment att = ats.member(0);
 ```
 
-By specific URI:
+By using a specific URI:
   
 ```java
 String attUri = "http://host/maximo/oslc/os/mxwodetail/_QkVERk9SRC8xMDAw/DOCLINKS/28";
 ```
 
-Using AttachmentSet
+By using AttachmentSet:
   
 ```java  
 Attachment att = ats.fetchMember(attUri);
 ```
 
-Or using MaximoConnector
+Or by using MaximoConnector:
   
 ```java 	
 Attachment att  = mc.attachment(attUri);
 ```
-* Delete the Attachment
+* Delete the attachment:
 
-Using Attachment (useful when you already have the attachment)
+By using the attachment, which is useful when you already have the attachment:
   
 ```java
 att.delete();
 ```
 
-Using MaximoConnector (useful when you just have the uri)
+By using MaximoConnector, which is useful when you have only the URI:
   
 ```java
 mc.deleteAttachment(attUri);
 ```
 
-## 3.6 Saved Query
-Maximo supports a feature called a Saved Query where a pre-built query for an application, such as Work Order Tracking, which allows a user to easily retrieve a common set of data (example: a list of Approved Work Order). Assuming public saved queries are present for  an application (like WOTRACK), you can use the savedQuery() api to query records based on defined filter criterion. We need also make sure that the authorized application is set accordingly for the Object Structure we are planning to use.
+## 3.6 Saved query
+Maximo Asset Management supports a feature that is called a Saved Query that is a pre-built query for an application, such as the Work Order Tracking application, which allows a user to easily retrieve a common set of data, for example, a list of approved work orders. Assuming public saved queries are present for an application, you can use the savedQuery() API to query records based on defined filter criterion. You also must ensure that the authorized application is set accordingly for the object structure you plan to use.
 
-Taking the "OWNER IS ME" query for WOTRACK (workorder) application as an example. Assuming MXWODETAIL Object Structure is setup with the "WOTRACK" as authorized application.
+Take the "OWNER IS ME" query for the Work Order Tracking (WOTRACK) application as an example, which assumes that the MXWODETAIL object structure is setup with "WOTRACK" as the authorized application.
 
-* Query the data
+* Query the data:
 
 ```java
 ResourceSet rs = mc.resourceSet("mxwodetail").savedQuery(new SavedQuery().name("WOTRACK:OWNER IS ME")).select("*").fetch();
 ```
 
-The select("*") queryes all attributes for the filtered set of mxwodetail. As mentioned earlier, we can do a partial resource selection like select("wonum","status").
+The select("*") statement queries all attributes for the filtered set of mxwodetail. As mentioned earlier, you can do a partial resource selection, such as select("wonum","status").
```

Comment:
line 771
the lone * is causing the rest of the paragraph to italicise the whole paragraph - is there something you can do to get Guthub to ignore the formatting?